ui/app/src/views/projects/RecentlyViewedDashboards.tsx

```diff
@@ -27,14 +27,23 @@ export function RecentlyViewedDashboards(props: RecentlyViewedDashboardsProps) {
 
   return (
     <Box id={props.id}>
-      <Stack direction="row" alignItems="center" justifyContent="space-between">
+      <Stack
+        sx={{
+          borderBottom: 1,
+          borderColor: 'divider',
+          width: '100%',
+          height: '72px',
```

Comment:
@Gladorme any idea about this one as well 😄 ?

Reply:
I was checking, but I think your solution is good 😄